ui-components/src/blocks/AppStatusItem.tsx

```diff
@@ -45,25 +44,22 @@ const AppStatusItem = (props: AppStatusItemProps) => {
                 {t("application.status")}: {t("application.statuses.submitted")}
               </span>
             </p>
-            <a href={`application/${application.id}`} className="button is-small">
+            <a href={props.applicationURL} className="button is-small">
               {t("application.viewApplication")}
             </a>
           </div>
         </section>
 
         <footer className="status-item__footer">
           <div className="status-item_links">
-            <LocalizedLink
-              className="status-item__link lined"
-              href={`/listing/${listing.id}/${listing.urlSlug}`}
-            >
+            <LocalizedLink className="status-item__link lined" href={props.listingURL}>
```

Comment:
FYI, LocalizedLink is deprecated and we should probably remove it soon. Best to use `LinkComponent` from the `NavigationContext` instead. (See the `LinkButton` component for example usage.)